sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -254,7 +254,7 @@ public String getDefaultLanguage() {
      *
      * For a list of supported entity types, check: <a href="https://aka.ms/taner">this</a>.
      * For a list of enabled languages, check: <a href="https://aka.ms/talangs">this</a>.
-     * This method will use the default language that sets up in
+     * This method will use the default language that is set using
```

Comment:
that is set using .... is not clear what you mean. consider rewording

Reply:
updated.